sop-website/docs/Waveform-Data/Waveform-Extension.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Waveform Extension
+id: Waveform-Extension
+description:  An SOP for the waveform extension to the OMOP CDM
+---
+
+## 1. Purpose
+
+This SOP provides detailed implementation guidance for the OMOP CDM Waveform Extension, encompassing specifications and updated procedures for the following tables: waveform_occurrence, waveform_registry, waveform_feature, and waveform_channel_metadata. It serves as an addendum to the previously published Multimodal Linkage SOP [https://chorus-ai.github.io/Chorus_SOP/docs/Multimodal-Linkage/], which first introduced the waveform_registry table. This addendum modifies and expands the scope of that guidance to reflect a broader, semantically integrated model for representing waveform acquisition events, derived features, and signal metadata. The updated schema enables consistent ingestion, standardization, and temporal alignment of physiological waveform data (e.g., ECG, EEG, ABP), supporting use cases in critical care, AI model development, and observational research within the OMOP CDM ecosystem.
+
+## 2. Scope
+
+Applicable to all ETL pipelines and data engineers responsible for transforming raw physiological waveform files into OMOP CDM via harmonized linkage tables.
+
+## 3. Procedural Steps
+
+### 3A. Understand The Table Population Order & Rationale
+
+The waveform_occurrence table must be populated first, as it defines the core acquisition event and provides a semantic and temporal anchor for all other tables. Each waveform file (waveform_registry), feature (waveform_feature), and channel metadata (waveform_channel_metadata) must link back to this acquisition context.
+
+| Step |           Table           |                              Rationale                              |
+|:----:|:-------------------------:|:-------------------------------------------------------------------:|
+| 1    | waveform_occurrence       | Establishes clinical and temporal context for the recording session |
+| 2    | waveform_registry         | Registers each raw waveform file, linked to the occurrence          |
+| 3    | waveform_channel_metadata | Describes per-signal-channel metadata for each registered file      |
+| 4    | waveform_feature          | Stores derived features from specific waveform-channel combinations |
+
+## 4.  Scan & Detect
```

Comment:
For consistency with the sections below should we change Scan & Detect to Populate Waveform Occurrence? Alternatively we could leave this as Scan & Detect and have a subsection (4A, 4B, etc.) for each table.